src/simple_bridge_util.erl

```diff
@@ -371,20 +371,20 @@ create_cookie_header(#cookie{name=Name, value=Value, max_age=MaxAge,
         create_cookie_path(Path),
         create_cookie_http_only(HttpOnly)
     ],
-    {"Set-Cookie", HeaderVal}.
```

Comment:
This particular change caused me a bit of headache with inets, as inets expects the headers to be strings (the header itself, not the header value).  Not a big deal - I was able to work around it.  But worth noting how these little things can have cascading effects.  Inets seemed to be the only one affected, so this behavior was safe to keep.

Reply:
ok